digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -110,24 +118,27 @@ public void run(TaskRequest request)
     {
         long taskId = request.getTaskId();
 
-        // set task name to thread name so that logger shows it
-        try (SetThreadName threadName = new SetThreadName(request.getTaskName())) {
-            try (TaskLogger taskLogger = callback.newTaskLogger(request)) {
-                TaskContextLogging.enter(LogLevel.DEBUG, taskLogger);
-                try {
-                    runningTaskMap.put(taskId, request);
+        tm.begin(() -> {
```

Comment:
👍 
BTW, I don't get this existing comment `use the HTTP request time as the runTime`. I have no idea what `the HTTP request` exactly means...
https://gist.github.com/frsyuki/7c62437dd55cefa0b343dcf593d8630e#file-task-callback-transaction-diff-L201